src/ecalc/cli/tests/snapshots/test_app/test_json_true/test_v3.json

```diff
@@ -3743,6 +3743,72 @@
           0.0
         ]
       },
+      "requested_inlet_pressure": {
+        "1900-01-01T00:00:00": {
```

Comment:
date is wrong ...

Reply:
or, I dont understand this date. is it expected?

Reply:
Think you are right, probably some date format issue when extracting the time value from the temporal model.

Reply:
1900 is the "default", when no date is given, ie that it is valid since "beginning of time", whatever that is. that is "behind the scenes" in general. If temporal models had been used, then other dates would show I think. Nevertheless, as we discuss other places here, probably makes sense to "flatten out" the temporal models to one dimensional timeseries ...

Reply:
The other properties should behave equally, so not sure why this one seems different...might be a bug? See e.g. recirculation_loss, why is there no 1900 timestamp there?

Reply:
.Temporal models are now merged, hence the date format is not an issue